armor62_decrypt.go

```diff
@@ -23,12 +23,12 @@ var (
 // as necessary. Returns the MessageKeyInfo recovered during header
 // processing, an io.Reader stream from which you can read the plaintext, the armor branding, and
 // maybe an error if there was a failure.
-func NewDearmor62DecryptStream(versionValidator VersionValidator, ciphertext io.Reader, kr Keyring) (*MessageKeyInfo, io.Reader, string, error) {
+func NewDearmor62DecryptStream(versionValidator VersionValidator, ciphertext io.Reader, kr Keyring) (key *MessageKeyInfo, vs io.Reader, brand string, err error) {
```

Comment:
Minor: what does `vs` stand for? Also the `key` is only referred to as `mki` in the code, you could change that too.